cyences_app_for_splunk/bin/cs_product_list.py

```diff
@@ -468,6 +472,23 @@ def build_app_dependency_search(app_list):
             }
         ],
     },
+    {
+        "name": "Vulnerability",
+        "app_dependencies": [],
+        "metadata_count_search": '`cs_vulnerabilities_indexes` sourcetype="*vuln*" | stats count',
```

Comment:
Sourcetype may not contain *vuln*
We need to stick with what we are using in the datamodel, alert or dashboard.